docs/preview/02-Guides/shared-access-key-authentication-with-azure-key-vault-integration.md

```diff
@@ -0,0 +1,159 @@
+# Shared access key authentication with Azure Key Vault integration
+The Arcus shared access key authentication is an API security filter to easily and safely share the same access key across a set of API endpoints. This kind of authentication is very common and therefore a popular feature in the Arcus WebApi library.
+
+The API security filter makes use of the [Arcus secret store](https://security.arcus-azure.net/features/secret-store) to retrieve the access keys. This makes sure that the secrets are safely accessed in the application.
```

Comment:
```suggestion
The API security filter makes use of the [Arcus secret store](https://security.arcus-azure.net/features/secret-store) to retrieve the access keys. This menas that the access key is safely stored in a secret-store and easily retrieved using an `ISecretProvider` in the application.
```

Reply:
Hmm, maybe the part with the `iSecretProvider` is a bit much here in the introduction as this user guide doesn't interact with the secret store directly, only via the shared access key authentication filter/attribute.

Reply:
Changed in https://github.com/arcus-azure/arcus.webapi/pull/374/commits/7cfa8efcec6675f5657c4d7d2c33786b23db5996

Reply:
Removed this sentence as it doesn't bring any value.